source/standalone/tutorials/04_sensors/benchmark_cameras.py

```diff
@@ -0,0 +1,868 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+This script might help you determine how many cameras your system can realistically run
+at different desired settings. You can supply different task environments
+to inject cameras into, or just test a sample scene. Additionally,
+you can automatically find the maximum amount of cameras you can run a task with through the
+autotune functionality.
+
+.. code-block:: bash
+
+    # Usage with GUI
+    ./isaaclab.sh -p source/standalone/tutorials/04_sensors/benchmark_cameras.py -h
+
+    # Usage with headless
+    ./isaaclab.sh -p source/standalone/tutorials/04_sensors/benchmark_cameras.py -h --headless
+
+"""
+
+"""Launch Isaac Sim Simulator first."""
+
+import argparse
+from collections.abc import Callable
+
+from omni.isaac.lab.app import AppLauncher
+
+# parse the arguments
+args_cli = argparse.Namespace()
+
+parser = argparse.ArgumentParser(description="This script can help you benchmark how many cameras you could run.")
+
+"""
+The following arguments only need to be supplied for when one wishes
+to try injecting cameras into their environment, and automatically determining
+the maximum camera count.
+"""
+parser.add_argument(
+    "--task",
+    type=str,
+    default=None,
+    required=False,
+    help="Supply this argument to spawn cameras within an known manager-based task environment.",
+)
+
+parser.add_argument(
+    "--autotune",
+    default=False,
+    action="store_true",
+    help=(
+        "Autotuning is only supported for provided task environments."
+        " Supply this argument to increase the number of environments until a desired threshold is reached."
+        "Install pynvml in your environment; ./isaaclab.sh -m pip install pynvml"
+    ),
+)
+
+parser.add_argument(
+    "--task_num_cameras_per_env",
+    type=int,
+    default=1,
+    help="The number of cameras per environment to use when using a known task.",
+)
+
+parser.add_argument(
+    "--use_fabric", action="store_true", default=False, help="Enable fabric and use USD I/O operations."
```

Comment:
Is this argument used for anything in the script?
I believe we can remove this no?

Reply:
It's used within `inject_cameras_into_task` when using a known environment. I figured it was good to include it for the sake of completeness as the training scripts also use this variable and it seemed like it impacted performance in my tests